tests/lldp/test_lldp.py

```diff
@@ -17,13 +17,13 @@ def lldp_setup(duthosts, rand_one_dut_hostname, patch_lldpctl, unpatch_lldpctl,
     unpatch_lldpctl(localhost, duthost)
 
 
-def test_lldp(duthosts, rand_one_dut_hostname, localhost, collect_techsupport):
+def test_lldp(duthosts, rand_one_dut_hostname, localhost, collect_techsupport, enum_frontend_asic_index):
     """ verify the LLDP message on DUT """
     duthost = duthosts[rand_one_dut_hostname]
 
-    config_facts  = duthost.config_facts(host=duthost.hostname, source="running")['ansible_facts']
-    lldpctl_facts = duthost.lldpctl_facts(skip_interface_pattern_list=["eth0"])['ansible_facts']
-
+    namespace = duthost.get_namespace_from_asic_id(enum_frontend_asic_index)
+    config_facts = duthost.config_facts(host=duthost.hostname, source="running",namespace=namespace)['ansible_facts']
```

Comment:
can we use the sonicasic.config_facts here ?

Reply:
@arlakshm Updated.